.github/workflows/bindings-wallet-nodejs.yml

```diff
@@ -45,14 +45,23 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        # os: [windows-latest, macos-latest, ubuntu-latest]
-        os: [windows-latest, ubuntu-latest]
+        os: [windows-latest, macos-13, ubuntu-latest]
         node: ["18"]
 
     steps:
       - name: Checkout the Source Code
         uses: actions/checkout@v3
 
+      - name: Select Xcode (macOS)
+        uses: maxim-lobanov/setup-xcode@v1
+        if: matrix.os == 'macos-latest'
```

Comment:
Just checking, is this supposed to be `macos-latest` ?

Reply:
It's not, good catch!